packages/website/src/pages/index.tsx

```diff
@@ -107,9 +108,13 @@ const features: FeatureItem[] = [
 
 function Feature({ title, description }: FeatureItem): JSX.Element {
   return (
-    <div className="col col--12 padding-vert--lg">
-      <h2 className="text--center">{title}</h2>
-      {description}
+    <div className="col col--12 padding-vert--lg ">
```

Comment:
Nit: lil extra space added

```suggestion
    <div className="col col--12 padding-vert--lg">
```

Reply:
I fixed it!